walletdb/bdb/db.go

```diff
@@ -9,7 +9,7 @@ import (
 	"os"
 
 	"github.com/btcsuite/btcwallet/walletdb"
-	"go.etcd.io/bbolt"
+	"github.com/coreos/bbolt"
```

Comment:
I think this needs to be the other path?

Reply:
I reverted the commit this time and this was part of it. Seems to build just fine.